integrationTests/testConsensusNode.go

```diff
@@ -326,7 +326,7 @@ func (tcn *TestConsensusNode) initNode(args ArgsTestConsensusNode) {
 	argsKeysHolder := keysManagement.ArgsManagedPeersHolder{
 		KeyGenerator:          args.KeyGen,
 		P2PKeyGenerator:       args.P2PKeyGen,
-		MaxRoundsOfInactivity: 10,
+		MaxRoundsOfInactivity: 0,
```

Comment:
is this needed? if 0, I think the main machine would not do anything, but the backup will take over

Reply:
for main machine i think is has to be zero, otherwise it will start as redundant node, and after 10 rounds it will take over